test/units/testsuite-74.bootctl.sh

```diff
@@ -266,8 +272,11 @@ EOF
 testcase_bootctl_varlink() {
     varlinkctl call --collect /run/systemd/io.systemd.BootControl io.systemd.BootControl.ListBootEntries '{}'
 
-    # We have no UEFI in the test environment, hence just check that this fails cleanly
-    ( SYSTEMD_LOG_TARGET=console varlinkctl call --json=short /run/systemd/io.systemd.BootControl io.systemd.BootControl.GetRebootToFirmware '{}' 2>&1 || true ) | grep -q io.systemd.BootControl.RebootToFirmwareNotSupported
+    # We may not have UEFI in the test environment
+    # if we don't we want to test whether it fails cleanly.
```

Comment:
This message is weirdly structured now? I don't get what you're trying to say here

Reply:
Is this clearer?

> We may have UEFI in the test environment.
> If we don't have UEFI then we can test whether bootctl's varlink API fails cleanly.
> If we do have UEFI then the rest of the clean fail tests should be skipped.